servicecomb-0.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,58 @@
+registryItems:
+  applications:
+  - {servicecomb-provider: nq 0}
+  - {servicecomb-consumer: nq 0}
+  instances:
+  - {servicecomb-provider: 1}
+  - {servicecomb-consumer: 1}
+  operationNames:
+  - dubbo-consumer: [sayHi]
+  - dubbo-provider: [sayHi]
```

Comment:
I think the operationName should be `org.apache.skywaking.apm.testcase.servicecomb.schemma.Hello. sayHi`